exporter/googlecloudexporter/README.md

```diff
@@ -274,6 +298,9 @@ processors:
       - delete_key(attributes, "instrumentation_version")
 ```
 
+**Note** It is not recommended to use these transformations with the googlecloud
+exporter in a logging pipeline.
```

Comment:
From the [referenced issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27803):
```
 We should make clear that the documentation does not apply to logging or trace, and that it should (probably) only be used with the prometheus receiver.
```
Should we expand this message in the README to include this, or did I misunderstand the context?

Reply:
This particular part of the documentation should be not recommended for logging or trace.  Added trace to the statement.